packages/core/src/components/input/input.stories.ts

```diff
@@ -0,0 +1,67 @@
+import { htmlMatrix, Meta, omit, Story } from '../../../../../docs';
+import { Input, InputProps } from './input.story';
+
+const disabled = [true, false] as const;
+const invalid = [true, false] as const;
+
+const type = [
+  'text',
+  'number',
+  'email',
+  'password',
+  'tel',
+  'url',
+  'search',
+] as const;
+type.toString = () => type.map((value) => `"${value}"`).join('|');
+
+export default {
+  title: 'Core/Input',
+  component: Input,
+  argTypes: {
+    disabled: {
+      table: { type: { summary: 'boolean' } },
+    },
+    invalid: {
+      table: { type: { summary: 'boolean' } },
+    },
+    placeholder: {
+      table: { type: { summary: 'string' } },
+    },
```

Comment:
🤓  [no action]
I'm still negatively surprised `docgen` can't pick any of these up :/

Reply:
It can, but not fully.

For example, it does not indicate that the prop is a "string":

![Screenshot 2021-03-05 at 12 41 11](https://user-images.githubusercontent.com/20243687/110116894-1382fe00-7db0-11eb-9387-75c55cb68b4e.png)

Which I dislike...

Also, otherwise not possible to achieve alphabetical order.

Reply:
Yea we have to figure that out at some point.